webapp/src/main/webapp/templates/freemarker/edit/forms/addAttendeeRoleToPerson.ftl

```diff
@@ -24,5 +24,14 @@ roleExamples-->
 <#assign acMultipleTypes = "'true'" />
 <#assign acTypes = "{activity: 'http://vivoweb.org/ontology/core#EventSeries,http://purl.org/NET/c4dm/event.owl#Event'}" />
 
+<#assign roleCode="Attendee">
+
+<#assign editTitle = "${i18n().edit_entry_for_attendee_role}"/>
+
+<#assign createTitle = "${i18n().create_entry_for_attendee_role}"/>
+
+<#assign nameLabel = "${i18n().institution_name}"/>
+
+<#assign  roleInLabel = "${i18n().role_in_institution}"/>
 <#--Each of the two stage forms will include the form below-->
-<#include "addRoleToPersonTwoStage.ftl">
+<#include "addRoleToPersonTwoStage.ftl">
```

Comment:
It would make me happy if you added back in the newline at the end so the little strikethrough and arrow go away :)